CONTRIBUTING.md

```diff
@@ -37,9 +36,9 @@ can be applied as quickly as possible:
    modify the `README.md` file to reflect that. Again, if you don't
    update the `README`, we have to, and this holds up acceptance.
 
-4. **Update the Changelog (`CHANGES`):** The changelog helps give an
+4. **Update the changelog (`CHANGES`):** The changelog helps give an
```

Comment:
This is the only change that caught my eye.

I'm still trying to figure out when it's a Changelog, when to call it a change log, and when to refer to the actual CHANGELOG.

http://keepachangelog.com/

Thoughts?